packages/forms/src/index.tsx

```diff
@@ -1,3 +1,49 @@
+/**
```

Comment:
Not important, but what would you say to breaking this mëga file up?

Maybe have something like:
- errors.ts
- inputs.tsx
- hooks.ts

Reply:
It is getting quite long, especially with all the comments I added. I'm sure I'll have to update this file again shortly so I'll think about it then!